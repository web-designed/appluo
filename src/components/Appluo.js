import React from 'react'
import EventsList from './EventsList'

class Appluo extends React.Component {
   render() {
      return (
         <div>
            <EventsList />
         </div>
      )
   }
}

export default Appluo