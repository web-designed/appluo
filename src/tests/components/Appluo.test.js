import React from 'react'
import { shallow } from 'enzyme'
import Appluo from '../../components/Appluo'

test('should render Appluo correctly', () => {
   const wrapper = shallow(<Appluo />)
   expect(wrapper).toMatchSnapshot()
})