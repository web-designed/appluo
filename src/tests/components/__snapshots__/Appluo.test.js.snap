// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Appluo correctly 1`] = `
<div>
  <Connect(EventsList) />
</div>
`;
